CONTRIBUTING.md

```diff
@@ -21,19 +20,17 @@ contributing, please review the guidelines and best practices below.
     *  [Testing](#testing)
  * [Community Guidelines](#community-guidelines)
 
-## Core API vs. Language Library
+## Providing Feedback
 
 This repository is specifically for the Google Ads API Client Library for Java. If you have feedback
-related to the core API, please submit it
-[here](https://developers.google.com/google-ads/api/support).
-
-## Providing Feedback
+related to the Google Ads API, please submit it by clicking the "Send feedback" button in the top right
```

Comment:
nit: Put **Send feedback** in bold. This is the [standard on developers.google.com for any UI elements](https://developers.google.com/style/ui-elements#formatting).